CMakeLists.txt

```diff
@@ -278,6 +278,16 @@ elseif(UNIX)
     target_compile_definitions(PortAudio PRIVATE PA_USE_COREAUDIO=1)
     set(PKGCONFIG_LDFLAGS_PRIVATE
       "${PKGCONFIG_LDFLAGS_PRIVATE} -framework CoreAudio -framework AudioToolbox -framework AudioUnit -framework CoreFoundation -framework CoreServices")
+  elseif(ANDROID)
+      find_package(OpenSLES)
+      cmake_dependent_option(OPENSLES "Enable support for OpenSLES" ON OPENSLES_FOUND OFF)
+
+      if(OPENSLES)
+        target_link_libraries(PortAudio PRIVATE "${OPENSLES_LIBRARIES}")
```

Comment:
Export an INTERFACE target from the find module rather than directly using `${OPENSLES_LIBRARIES}`